### PR TITLE
etcdmain: add missing flag doc

### PR DESCRIPTION
**Pull request by gyuho**
/cc @heyitsanthony @xiang90

### CODE REVIEW

File: etcdmain/help.go [hunk 1]
```diff
@@ -116,6 +116,8 @@ security flags:
 		enable peer client cert authentication.
 	--peer-trusted-ca-file ''
 		path to the peer server TLS trusted CA file.
+	--peer-auto-tls 'false'
+		peer TLS using generated certificates.
```
Comment (xiang90) [inline etcdmain/help.go]: peer TLS using generated certificates if --peer-key-file and --peer-cert-file are not provided.

**Comment by gyuho** [comment]
@xiang90 Just fixed. PTAL. Thanks.

**Comment by xiang90** [comment]
lgtm. defer to @heyitsanthony

File: etcdmain/help.go [hunk 2]
```diff
@@ -116,6 +116,8 @@ security flags:
 		enable peer client cert authentication.
 	--peer-trusted-ca-file ''
 		path to the peer server TLS trusted CA file.
+	--peer-auto-tls 'false'
+		peer TLS using generated certificates if --peer-key-file and --peer-cert-file are not provided.
```
Comment (heyitsanthony) [inline etcdmain/help.go]: `self-generated`?

**Comment by gyuho** [comment]
@heyitsanthony PTAL. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by xiang90** [comment]
@gyuho CI fails. Can you create a new issue for the failure case?

**Comment by gyuho** [comment]
sure.

**Pull request merged**